src/app/db/migrations/versions/20260527_1540_create_ai_suggestions_schema.py

```
@@ -0,0 +1,121 @@
+"""create ai_suggestions schema
+
+Revision ID: d4a2c8a0705b
+Revises: 20260512_0900
+Create Date: 2026-05-27 15:40:00.260749+00:00
+
+"""
+
+from typing import Sequence, Union
```

Ruff (0.15.17)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)